app.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_people
+    puts 'List of People:'
+    @people.each { |person| puts "ID: #{person.id}, #{person.class.name}: #{person.name}" }
+  end
+
+  def list_books
+    puts 'List of Books:'
+    @books.each_with_index { |book, index| puts "#{index}. #{book.title} by #{book.author}" }
+  end
+
+  def create_person
+    puts "Enter the person's name:"
+    name = gets.chomp
+
+    puts "Enter the person's age:"
+    age = gets.chomp.to_i
+
+    puts 'Is the person a student? (yes/no):'
+    is_student = gets.chomp.downcase
+
+    if is_student == 'yes'
+      puts 'Does the student have parent permission? (yes/no):'
+      parent_permission = gets.chomp.downcase == 'yes'
```

Comment:
- You did a great job by implementing the required changes from the previous reviewer however the app is not working as expected, please make sure that you are able to create a new person as you see in the picture below it brings an error when I'm trying to create a new person. please fix that :+1: 

![image](https://github.com/berylcl/OOP-school-library/assets/95297251/926d71cb-b822-4ec1-9113-b495c7b1cf09)